Add blood donation form (#47)
Dibasic/SS13PaperworkSimulator: templates/medical/blood_donation.txt

> [center][b]Blood Donation Application Form[/b][/center]

Reviewer: Would rephrase to "blood donation informed consent form" or something, "application" isn't quite the word for the process imo

[hr]

> First time donor? [field] [small](yes/no)[/small][br]

Reviewer: ```suggestion
Have you donated blood before? [field] [small](yes/no)[/small][br]
```

[hr]

> [u]For Repeat Donor[/u][br]

Reviewer: ```suggestion
[u]For Repeat Donors[/u][br]
```

Did you encounter any problems in your last donation? [field]
[small](No problems / fainting / bruise / difficulties finding vein / other)[/small][br]
[hr]

> Blood group: [field][br]

Reviewer: ```suggestion
Blood type: [field][br]
```

Actually when I've given blood they've tested it after sending it to their lab, they wouldn't rely on the donor to know their blood type.

> Age: [field][i]year[/i] Sex: [field] Weight: [field][i]kg[/i][br]

Reviewer: ```suggestion
Date of Birth: [field] Sex: [field] Height: [field] Weight: [field][i]kg[/i][br]
```

Reviewer: Date of birth on forms is just a personal preference, if you keep 'age' then I would put 'years' instead of 'year'

> If more than 60-70 years old, Do you have medical certificate? [field] [small](yes/no)[/small][br]

Reviewer: Not certain what this line means, may want to rephrase. Also, give a single age, not a range

Author: Took this from a form I found online.

Reviewer: I would adjust the age still.
```suggestion
If more than 60 years old, do you have medical certificate? [field] [small](yes/no)[/small][br]
```

Occupation: [field][br]
Name: [i]Mr. / Ms. / Mrs.[/i] [field] Signature: [field][br]
[hr]
[hr]
[center][b]For your own safety and the safety of the patient who will receive your blood, please answer the following questions, if applicable, to the best of your knowledge:[/b][/center]
[hr]

> [u]Category 1 [small](For women only)[/small][/u][br]

Reviewer: ```suggestion
[u]Obstetric History[small](if applicable)[/small][/u][br]
```

> 1. Pregnant? [field][br]

Reviewer: ```suggestion
1. Are you now or do you plan to become pregnant? [field][br]
```

> [u]Obstetric History[small](if applicable)[/small][/u][br]

Reviewer: If we're leaving "Category 2" and "Category 3," may as well label this one "Category 1"
```suggestion
Category 1: [u]Obstetric History[small](if applicable)[/small][/u][br]
```

> 2. Do breast-feed? [field][br]

Reviewer: ```suggestion
2. Do you breastfeed? [field][br]
```

> 3. Gave birth or miscarriage in the last 6 months? [field][br]

Reviewer: ```suggestion
3. Have you given birth or miscarried in the last 6 months? [field][br]
```

[hr]

> [u]Category 2[/u]

Reviewer: Is there a specific way this category is different from the others? If so, should label it more specifically

> 4. Had diarrhea in the last 7 days? [field][br]

Reviewer: ```suggestion
4. Have you had diarrhea in the last 7 days? [field][br]
```
suggesting present perfect for all of these

> 5. Had major surgery in the last 6 months or minor surgery in the last 7 days? [field][br]

Reviewer: ```suggestion
5. Have you undergone any surgery in the last 6 months? [field][br]
```

I don't know that they'd trust a patient to judge whether or not their surgery was major

> 6. Do you drink alcohol or others? [field][br]

Reviewer: ```suggestion
6. Do you drink alcohol? [field] If so, how many drinks do you have in a typical week? [field][br]
```
something like this

> 7. Had a history of drug use or had you been improsoned in the last 3 years? [field][br]

Reviewer: ```suggestion
7. Do you have any history of drug addiction or have you been imprisoned in the last 3 years? [field][br]
```

> 8. Had a blood transfusion in the past 1 year? [field][br]

Reviewer: ```suggestion
8. Have you had a blood transfusion in the past 12 months? [field][br]
```

> [u]Category 3[/u]

Reviewer: Is there a specific way this category is different from the others? If so, should label it more specifically

> 9. Have menstruation? [small](to be answered by female only)[/small] [field][br]

Reviewer: ```suggestion
9. Are you menstruating? [small](if applicable)[/small] [field][br]
```

> 10. Do you feel fit enough and have enough rest last night? [field][br]

Reviewer: ```suggestion
10. Do you feel fit and well rested? [field][br]
```
might be awkward

> 11. Did you take any medicine / antibiotics? [field][br]

Reviewer: ```suggestion
11. What medication have you taken in the last 7 days? [field][br]
```

> 12. Have you or any of your family member ever had hepatitis? [field][br]

Reviewer: ```suggestion
12. Do you have any history or family history of hepatitis? [field][br]
```

> 13. Have asthma, epilepsy, chronic skin disease, chronic cough, tuberculosis, allergies, high blood pressure, heart/kidney/thyroid disease, cancer, bleeding disorder, etc? [field][br]

Reviewer: ```suggestion
13. Do you have asthma, epilepsy, chronic skin disease, chronic cough, tuberculosis, allergies, high blood pressure, heart/kidney/thyroid disease, cancer, any bleeding disorder, or any communicable disease? [small](please list which, if any, apply)[/small] [field][br]
```

> 14. Have ear/body piercings, tattoos made or removed or acupuncture? [field][br]

Reviewer: ```suggestion
14. Have you had any body piercing, any tattoo applied or removed, or any acupuncture in the last 12 months? [field][br]
```
something like this

> 15. Did you get any vaccinations in the last 2 months? [field][br]

Reviewer: ```suggestion
15. Have you had any vaccinations in the last 2 months? [field][br]
```

> 26. Others: [field][br]

Reviewer: ```suggestion
16. Other notes or issues: [field][br]
```

[hr]

> [small]I hereby certify that I have answered the following questions truthfully and that, to the best of my knowledge, my blood is safe for donation.

Reviewer: ```suggestion
[small]I certify that I have answered the above questions truthfully and that, to the best of my knowledge, my blood is safe for donation.
```

> I have been informed that my blood will be tested for syphilis, hepatitis B and C, as well as HIV/AIDS.

Reviewer: ```suggestion
I have been informed that my blood will be tested for syphilis, hepatitis B and C, HIV, and other bloodborne illnesses.
```

> I hereby voluntarily donate blood without expecting any type of remuneration.

Reviewer: ```suggestion
I consent to donate blood without expecting any type of remuneration.
```

> The blood may be given to any patient or for research purpose as deemed suitable by Torch medical personnel.

Reviewer: ```suggestion
I accept that the blood I donate may be given to any patient or used for research purposes as deemed suitable.
```
something like this, also like making this non-Torch-specific

> I certify that the Torch medical staff are not responsible for any untoward effects that may occour after this blood donation.

Reviewer: ```suggestion
I agree that the staff and technicians involved in my donations are not liable for any untoward effects that may occur after my donation.
```
Just changing the voice of this, also would remove Torch but that's just my opinion

> I shall be pleased to donate blood again.[/small]

Reviewer: Not sure what this means

Author: Took this from a form I found online.

Reviewer: It sounds weird to me, I'd suggest taking out this one line.
```suggestion
[/small]
```

Donor signature: [field][br]

> Doctor/Staff signature: [field][br]

Reviewer: Not certain that the nurse involved needs to sign, this form may only need the donor's informed consent
